ui/src/components/customModal/ModifyUnitDetailModal.tsx

```diff
@@ -501,30 +495,30 @@ export default function ModifyUnitDetailModal({
             </Form.Item>
           </Col>
           <Col span={8}>
-            <Row gutter={24}>
-              <Col>
-                <Form.Item label="min iops" name={['unitConfig', 'minIops']}>
-                  <InputNumber
-                    min={minResource.minIops}
-                    placeholder={intl.formatMessage({
-                      id: 'Dashboard.components.customModal.ModifyUnitDetailModal.PleaseEnter',
-                      defaultMessage: '请输入',
-                    })}
-                  />
-                </Form.Item>
-              </Col>
-              <Col>
-                <Form.Item label="max iops" name={['unitConfig', 'maxIops']}>
-                  <InputNumber
-                    min={minResource.maxIops}
-                    placeholder={intl.formatMessage({
-                      id: 'Dashboard.components.customModal.ModifyUnitDetailModal.PleaseEnter',
-                      defaultMessage: '请输入',
-                    })}
-                  />
-                </Form.Item>
-              </Col>
-            </Row>
+            <Form.Item label="min iops" name={['unitConfig', 'minIops']}>
+              <InputNumber
+                style={{ width: 217 }}
+                min={minResource.minIops}
+                max={MAX_IOPS}
+                placeholder={intl.formatMessage({
+                  id: 'Dashboard.components.customModal.ModifyUnitDetailModal.PleaseEnter',
+                  defaultMessage: '请输入',
+                })}
+              />
+            </Form.Item>
+          </Col>
+          <Col span={8}>
+            <Form.Item label="max iops" name={['unitConfig', 'maxIops']}>
+              <InputNumber
+                style={{ width: 222 }}
+                min={minResource.maxIops}
```

Comment:
Does this mean that `maxIops` can not be decreased?

Reply:
I remember there is a minimum value for resources (minResource). If it is less than this value, the resource pool cannot be created. So the input box here limits the minimum value.

Reply:
OK